Source/Core/Core/PowerPC/JitCommon/JitCache.h

```diff
@@ -66,7 +54,7 @@ class ValidBlockBitSet final
 {
 	enum
 	{
-		VALID_BLOCK_MASK_SIZE = 0x20000000 / 32,
+		VALID_BLOCK_MASK_SIZE = 0x100000000 / 32,
```

Comment:
This line in particular means the m_valid_block array takes up 16MB; that might be bigger than it needs to be, but making it smaller has complications.

If my computations are correct, this patch saves about 100MB of memory overall.